packages/design/src/Icon/Icon.jsx

```diff
@@ -111,6 +111,7 @@ export const ClipboardUser = makeFontIcon(
   'ClipboardUser',
   'icon-clipboard-user'
 );
+export const Clock = makeFontIcon('Clock', 'icon-clock3');
```

Comment:
Yes, I updated the icomoon lib when started working on this issue and since then the assets were updated on https://github.com/gravitational/webapps/pull/952/files#diff-cf3b0c544bd0e10329bf5ab41682df0c6156bfe9bee8096ef83ff7b0f865765e, that's why I didn't had to update the assets in this PR.

Reply:
i think it's ok to keep it